tools/editors-libraries.adoc

```diff
@@ -0,0 +1,177 @@
+* `unsubscribeToContentChanges(callback: (isDirty: boolean) => void): void`: Unsubscribes the passed callback from content changes.
+* `markAsSaved(): void`: This resets the editor state, signaling that its content is saved. It also fires the subscribed callbacks for content changes.
+* `undo(): void`: Undo the last change in the Editor. It will also fire the subscribed callbacks of content changes.
+* `redo(): void`: Redo the last undone change in the Editor. It will also fire the subscribed callbacks of content changes.
+* `close(): void`: Closes the Editor.
+* `envelopeApi: MessageBusClientApi<KogitoEditorEnvelopeApi>`: Advanced Editor API.
+* `canvas`: Canvas API that exposes methods to manipulate the canvas.
+ ** `getNodeIds(): Promise<string[]>`: Returns a Promise containing the ID attributes of all nodes displayed in the editor canvas.
+ ** `getBackgroundColor(uuid: string): Promise<string>`: Returns a Promise containing the background color of a node with the provided UUID.
+ ** `setBackgroundColor(uuid: string, backgroundColor: string): Promise<void>`: Sets the background color of a node with the provided UUID.
+ ** `getBorderColor(uuid: string): Promise<string>`: Returns a Promise containing the border color of a node with the provided UUID.
+ ** `setBorderColor(uuid: string, borderColor: string): Promise<void>`: Sets the border color of a node with provided UUID.
+ ** `getLocation(uuid: string): Promise<number[]>`: Returns a Promise containing the canvas location of a node with the provided UUID.
+ ** `getAbsoluteLocation(uuid: string): Promise<number[]>`: Returns a Promise containing the window location for a node with the provided UUID.
+ ** `getDimensions(uuid: string): Promise<number[]>`: Returns a Promise containing the dimensions of a node with provided UUID.
+ ** `applyState(uuid: string, state: string): Promise<void>`: Applies state to a node given the UUID [None, Selected, Highlight, Invalid].
+ ** `centerNode(uuid: string): Promise<void>`: Centers node on viewable Canvas.
+
+== New DMN Editor
+
+The modern DMN Editor, built with React, is faster and supports up to DMN 1.5 specifications while maintaining the same simple-to-use API.
+
+.New DMN Editor
+image::tools/editorsLibraries/new-dmn-editor.png[]
+
+=== Installing and using the new DMN Editor
+
+First, add the library to the project's `package.json`:
+[source,shell,subs="+attributes"]
+----
+npm install @ibm/@ibm/bamoe-standalone-dmn-editor@{NPM_VERSION}
+----
+
+Import the editor library to a JS file with:
+[source,javascript,subs="+attributes"]
+----
+import * as NewDmnEditor from "@ibm/bamoe-standalone-dmn-editor/dist"
+----
+
+In the HTML, add a `div` container for each Editor; that's where the library will load an iFrame with all the Editor's resources:
```

Comment:
In the HTML, add a 'div' container for each editor where the library loads an iFrame with all the editor's resources: